lib/rubocop/cop/capybara/specific_matcher.rb

```diff
@@ -23,7 +23,7 @@ module Capybara
       #   expect(page).to have_no_link('foo', class: 'cls', href: 'http://example.com')
       #   expect(page).to have_table(class: 'cls')
       #   expect(page).to have_select
-      #   expect(page).to have_field('foo')
+      #   expect(page).to have_field(with: 'foo')
```

Comment:
question) Example to verify that the name attribute has an input field of 'foo', is there a problem with the code in the original example?

Reply:
It's not exactly a problem per se, it's just that all the rest of the example correspond to a line in the [bad](https://github.com/rubocop/rubocop-capybara/blob/8b63fc80d26639fac81eb26d9e6cd692adbe81b0/lib/rubocop/cop/capybara/specific_matcher.rb#L10) part. Only the last one is different - and I think the intention was for it also to be the same - just the correct way to write it. Adding the `with:` does that - makes it with the same meaning as the "bad" example.